src/dawn/engine_dawn.cc

```diff
@@ -359,13 +356,30 @@ Result EngineDawn::DoClear(const ClearCommand* command) {
   color_attachment.clearColor = render_pipeline->clear_color_value;
   color_attachment.loadOp = ::dawn::LoadOp::Clear;
   color_attachment.storeOp = ::dawn::StoreOp::Store;
-
-  // Attach that colour attachment to the render pass.
   ::dawn::RenderPassColorAttachmentDescriptor* rpca[] = {&color_attachment};
+
+  // Then describe the depthStencil attachment, and how it is initialized
+  // via the load ops. Both load op are "clear" to the clear values.
+  ::dawn::RenderPassDepthStencilAttachmentDescriptor depthStencil_attachment =
+      ::dawn::RenderPassDepthStencilAttachmentDescriptor();
+  ::dawn::RenderPassDepthStencilAttachmentDescriptor* rpda = nullptr;
```

Comment:
Can we rename rpda to depth_stencil_descriptor? It's longer, but clearer as I keep forgetting what rpda stands for